evalml/tests/component_tests/test_text_featurizer.py

```diff
@@ -218,6 +218,26 @@ def test_lsa_primitive_output():
     assert_frame_equal(expected_features, features.to_dataframe(), atol=1e-3)
 
 
+def test_featurizer_custom_types(text_df):
+    # force one of the two provided columns to be a user-specified type.
+    # if the output contains text features for col_2, then the text featurizer didn't pass the right
+    # ww types to LSA, because LSA still thought col_2 was natural language even though the user said otherwise.
+    X = infer_feature_types(text_df, {"col_2": "categorical"})
+    tf = TextFeaturizer()
+    tf.fit(X)
+
+    expected_col_names = set(['col_2',
+                              'DIVERSITY_SCORE(col_1)',
+                              'LSA(col_1)[0]',
+                              'LSA(col_1)[1]',
+                              'MEAN_CHARACTERS_PER_WORD(col_1)',
+                              'POLARITY_SCORE(col_1)'])
+    X_t = tf.transform(X)
+    assert set(X_t.columns) == expected_col_names
+    assert len(X_t.columns) == 6
+    assert set(X_t.logical_types.values()) == {ww.logical_types.Double, ww.logical_types.Categorical}
```

Comment:
Are you able to explicitly just check the logical type of "col_2"?  or do an ordered match?  I think it would just be nice to see that the specific user specified type and column were the same before and after.  It would be a little clearer to test readers that need to go through and realize that all the other fields will be doubles.

Reply:
Oh yeah, that's true.
So, the fact that "col_2" did not end up being converted into text features implies that the text featurizer saw it as categorical-type, not as natlang type.
But you're right, the extra step would be to do an ordered match. I'll do it.